src/presentation/http/schema/OauthSchema.ts

```diff
@@ -0,0 +1,30 @@
+
+export const OauthSchema = {
+  $id: 'OauthSchema',
+  type: 'object',
+  required: [
+    'clientId',
+    'clientSecret',
+  ],
+  properties: {
+    clientId: {
+      type: 'string',
+      description: 'Google client id token',
+    },
+
+    clientSecret: {
+      type: 'string',
+      description: 'Google client secret key',
+    },
+
+    accessToken: {
+      type: 'string',
+      description: 'The returned access token from Google',
+    },
+
+    refreshToken: {
```

Comment:
access and refresh tokens are not related to oauth, its our custom fields. I'd suggest creating the AuthSession schema and move them there.